frontend/src/pages/EditUserPage.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { useEffect } from 'react';
+import {
+  Container,
+  Row,
+  Col,
+  Spinner,
+  Button,
+  Navbar,
+  FormControl,
+  FormGroup,
+  FormLabel,
+  InputGroup,
+  ButtonGroup,
+  ButtonToolbar,
+} from 'react-bootstrap';
+import { fetchUserDetail, getUpdateUserAction } from 'actionCreators/usersActionCreator';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from 'reducers/rootReducer';
+import { IUserDetails } from 'actions/adminActions';
+import _ from 'lodash';
+import { Formik, ErrorMessage, Form } from 'formik';
+import { UserUpdateSchema } from 'utils/YupSchema';
+import { IUserDetailParams } from 'constants/API';
+import { ILookupCodeState } from 'reducers/lookupCodeReducer';
+import { ILookupCode } from 'actions/lookupActions';
+import * as API from 'constants/API';
+import './EditUserPage.scss';
+
+const EditUserPage = (props: IUserDetailParams) => {
+  useEffect(() => {
+    dispatch(fetchUserDetail({ id: '41a46ea3-e815-4272-9a17-693ba2409841' }));
+  }, []);
```

Comment:
Currently hard-coded - but page is set up to take a user id prop, will gather the user details based on the prop passed.

Reply:
Is there a way to not hard-code this value?  Shouldn't it be part of either the path or query string params?

Reply:
Currently if I change it to props.id it will work, the component just needs to be passed the appropriate value. I guess I just need to figure out how we want to get to this page. I was thinking from the manage users page there is an edit button and on click would pass the corresponding prop